client/src/components/Profile/ProfileTabs.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import BasicTabs from "components/Tabs/BasicTabs";
+import styled from "styled-components";
+
+// Styled component named StyledButton
+const StyledTabs = styled.div`
+  .ant-tabs-ink-bar {
+    border-bottom: 2px solid #282828;
+    width: 200px;
+  }
+  .ant-tabs-tab-btn {
+    color: #939393;
+  }
+  .ant-tabs-tab-active > .ant-tabs-tab-btn {
+    font-family: Poppins;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 21px;
+    letter-spacing: 0px;
+    text-align: center;
+    color: #282828;
+  }
+  .ant-tabs-tab-disabled > .ant-tabs-tab-btn {
+    color: #e1e1e1;
+  }
+  .ant-tabs-nav-list {
+    margin: auto;
+    width: 800px;
+    display: flex;
+    justify-content: space-around;
+    overflow: visible;
+  }
+`;
+
+export default function ProfileTabs(props) {
```

Comment:
Why do we need a separate component for ProfileTab and one for BasicTab?

Reply:
BasicTabs creates a default ant-d styled tab, that could be used anywhere in the application (outside of the profile component) as long as you pass in the needed props. The ProfileTab components adds on the styling you need to make it look how we want it in the profile section. This seemed consistent with the pattern I saw elsewhere in FP.

So if someone wanted tabs for say the feed section, but they didn't want it black and they wanted it to be blue with some fancy color for the highlighted, they wouldn't have to make a whole new tabs component, just a similar styled version of BasicTabs